src/encord_active/lib/dataset/summary_utils.py

```diff
@@ -30,19 +30,62 @@ def get_median_value_of_2D_array(array: np.ndarray) -> np.ndarray:
     return array[item_index[0][0], :]
 
 
-def get_all_annotation_numbers(project_folder: Path) -> Tuple[int, int]:
+def get_all_annotation_numbers(project_folder: Path) -> dict[str, Union[dict, int]]:
     """
-    returns (number of classification label, number of object label)
+    returns label statistics for both objects and classifications. Does not count nested
+    labels, only counts the immediate labels.
     """
+    labels: dict = {"objects": {}, "classifications": {}}
     classification_label_counter = 0
     object_label_counter = 0
 
+    project_ontology = json.loads((project_folder / "ontology.json").read_text(encoding="utf-8"))
+    for object_item in project_ontology["objects"]:
+        labels["objects"][object_item["name"]] = 0
+    for classification_item in project_ontology["classifications"]:
+        labels["classifications"][classification_item["attributes"][0]["name"]] = {}
+
+        # For radio and checkbox types
```

Comment:
I think presenting a comprehensive view of the labels in the dashboard is better. This part is just for reporting and does not affect the other parts, so we should also report checklist and text type classifications as our customers are also using those.

Reply:
They are using those but won't be able to find them anywhere else in the app, which might be a bit confusing.

Reply:
I think it might be better if we present them otherwise there will be missing information and an incomplete view of the dataset/labels, but you might be right in terms of other places. Let's ask @frederik-encord for the final decision. Should we include class distribution charts for the checklist and text-type classifications?

Reply:
@frederik-encord

Reply:
I think that we should include them for the following reason. If users see their checkboxes in the summary dashboard but not elsewhere, hopefully, they will reach out to us and "pull" the remaining features around checkboxes out of us.